engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-details.html

```diff
@@ -63,7 +63,7 @@ <h5 translate>Copy</h5>
   </div>
 
   <nav>
-    <ul class="nav nav-tabs">
+    <ul class="nav nav-tabs" ng-show="contentView.$resolved">
```

Comment:
No, because the content view is not null as soon as this is called: https://github.com/Katello/katello/pull/5062/files#diff-9e2842b2f6bf4c5ce3edf49e34dce2e0R35